qt/scientific_interfaces/Muon/MuonAnalysis.cpp

```diff
@@ -2035,6 +2038,44 @@ void MuonAnalysis::selectMultiPeak(const QString &wsName) {
   selectMultiPeak(wsName, boost::optional<QString>());
 }
 
+/**
+* Enable PP tool for the plot of the given WS. Does not update data selector
+* @param wsName Name of the WS which plot PP tool will be attached to.
+* is for "load current run" where the data file has a temporary name like
+* MUSRauto_E.tmp
+*/
+void MuonAnalysis::selectMultiPeakNoUpdate(const QString &wsName) {
```

Comment:
A lot of the code in this method is repeated from `SelectMultiPeak`. I think it would be neater to have `SelectMultiPeak` call `SelectMultiPeakNoUpdate`.

Reply:
I would like that too. I think it would have to be two parts as the difference is in the middle of the code block and cannot be moved